src/carter-mc-barter-narter-is-dope.css

```diff
@@ -0,0 +1,11 @@
+body {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+div {
+  
```

Comment:
Why is there nothing here? This isn't Nam there are rules.

Reply:
Ah shiiii gud point

Reply:
1 sec, will RM 🔪